lib/api/controller/realtime.js

```diff
@@ -54,9 +53,18 @@ class RealtimeController extends NativeController {
     assertHasIndexAndCollection(request);
     assertHasBody(request);
 
+    if ( request.input.args.propagate === undefined
+      || request.input.args.propagate === null
+    ) {
+      request.input.args.propagate = true;
```

Comment:
Shouldn't we allow that option only for subscriptions made by plugins?
(we can limit this by using the value of the `request.context.connection.protocol` property)